dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.jakarta.connectors-2.0.internal.api.feature

```diff
@@ -1,5 +1,5 @@
 -include= ~${workspace}/cnf/resources/bnd/feature.props
-symbolicName=io.openliberty.jakarta.internal.api.connectors-2.0
```

Comment:
I am slightly confused by the name of this feature ... how can it be both internal and an API? The pattern used by most components is that a private feature to expose only a (Jakarta) API would simply be called `io.openliberty.jakarta.connectors-2.0`.